src/components/FilterToolbar/FilterToolbar.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { Level, LevelItem } from '@patternfly/react-core';
+import { Table, TableHeader, TableBody, ICell, IRow } from '@patternfly/react-table';
+import { FilterToolbar, FilterType, FilterCategory } from './FilterToolbar';
+import { useFilterState } from '../../hooks/useFilterState';
+
+export const FilterableTable: React.FunctionComponent = () => {
+  // import { Table, TableHeader, TableBody, ICell, IRow } from '@patternfly/react-table';
+
+  interface IFruit {
+    name: string;
+    price: string;
+    description: string;
+  }
+
+  const fruits: IFruit[] = [
+    { name: 'Apple', price: '$0.83', description: 'Red delicious!' },
```

Comment:
We can probably also drop the descriptions since we're not including them in this table